README.md

```diff
@@ -88,3 +88,34 @@ item.name = "Item"
 item.active = false
 item.save
 ```
+
+### Inheritance Support
+This support allows models to be extended. The child model must include `Aws::Record` 
```

Comment:
I think we can be a little clearer in the first sentence for a user that isn't familiar with aws-record.  Maybe something like:

"Aws Record models can be extended using standard ruby inheritance". (to make it clear what sort of model.  and that they can be extended using ruby inhitence rather than some other means).